command_helper/jump_to_file.py

```diff
@@ -3,10 +3,10 @@
 
 try:
     from parser_utils.file_formatter import lib_table_name
-    from noralize_cell import get_data_from_json
+    from normalize_cell import get_data_from_json
 except ImportError:
```

Comment:
Perhaps not in this commit, but if you are going to support only Python 3, then these type of try/except are not needed.

Reply:
Ok.